images/nginx/Makefile

```diff
@@ -22,7 +22,7 @@ INIT_BUILDX=$(DIR)/../../hack/init-buildx.sh
 
 # 0.0.0 shouldn't clobber any released builds
 SHORT_SHA ?=$(shell git rev-parse --short HEAD)
-TAG ?=v$(shell date +%Y%m%d)-$(SHORT_SHA)
+TAG ?=$(shell cat TAG)
```

Comment:
Tag is not on main anymore, maybe add a comment about that.

Reply:
Actually I didn't change anything in `images/nginx-1.25`. I only renamed it to `images/nginx`.

Also this `TAG` references `images/nginx/TAG` and we have this on every branch, right?